tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_1_tensorrt7_2

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------
```

Comment:
if we have cuda11_3, probably cuda11_1 isn't needed since 11.3 is compatible with all of 11.x

Reply:
we may want to move to 11_4 though to align with everything else